src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -125,6 +131,21 @@ public void initialize() {
     numberOfPlayersComboBox.getSelectionModel().select(0);
     numberOfPlayersComboBox.getSelectionModel().selectedItemProperty().addListener(observable -> loadLeaderboard());
 
+    leaderboardSearchTextField.textProperty().subscribe(s -> {
+      leaderboardTable.setItems(leaderboardFilteredList);
+    });
```

Comment:
The set items shouldn't be in a listener. It just needs to be called once in the initialize function